pkg/scheduler/framework/plugins/interpodaffinity/plugin.go

```diff
@@ -54,6 +55,15 @@ func (pl *InterPodAffinity) Name() string {
 	return Name
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable
+func (pl *InterPodAffinity) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		{Resource: framework.Pod, ActionType: framework.All},
```

Comment:
Can you reword https://github.com/kubernetes/kubernetes/pull/100020#discussion_r590612561 here? As the reasons are not trivial, we'd better leave in-line comments here.

Reply:
I've added inline comments as suggested with the latest push. Please let me know if anything else needs to be updated @Huang-Wei